packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -98,6 +99,9 @@ export class StripeWebhookHandler extends StripeHandler {
         case 'invoice.payment_failed':
           await this.handleInvoicePaymentFailedEvent(request, event);
           break;
+        case 'product.updated':
```

Comment:
I don't think this event is triggered when a plan is updated.

Reply:
I tested manually, you'll have to run `stripe listen --forward-to localhost:9000/v1/oauth/subscriptions/stripe/event` in order to have the events picked up locally.

Reply:
Yea, I'd expect the parent object to trigger. Same happens with most customer child objects triggering a customer.updated.

Reply:
At the time of my comment I verified it by simply editing a plan/price in the Stripe dashboard and the checked the events (since I didn't have Stripe CLI and everything else running) , and I didn't see a product.updated.  I just tried the same thing again and I still don't